boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -181,11 +181,21 @@
 	};
 };
 
-&spi3 {
-	status = "okay";
-	sck-pin = <41>;
-	mosi-pin = <42>;
-	miso-pin = <43>;
+arduino_serial: &uart1 {
+	status = "disabled";
+	compatible = "nordic,nrf-uart";
```

Comment:
`uart1` can be used only as UARTE, you need to remove this line.

Reply:
fixed